.github/workflows/kfp-kubernetes-test.yaml

```diff
@@ -0,0 +1,37 @@
+name: KFP K8s Platform Tests
```

Comment:
Can we name it `kfp-kubernetes library tests` to keep consistency with https://github.com/GoogleCloudPlatform/oss-test-infra/blob/3738b70f78b971cc281b1791cf8e73cfbe08ae55/prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml#L268?

Reply:
Done